README.md

```diff
@@ -133,7 +133,7 @@ cd h2o-3
 ./gradlew build -x test
 
 # Start H2O
-java -jar build/h2o.jar
+java -jar bin/build/h2o.jar
```

Comment:
This is incorrect - we are not producing `h2o.jar` into the folder `bin`.
See: https://github.com/h2oai/h2o-3/blob/master/h2o-assembly/build.gradle#L51 - it copies h2o.jar assembly into the parent `build/`:

```groovy
task copyJar(type: Copy) {
    from ("${project.buildDir}/libs"){
        include assembly
    }
    into "${project.parent.buildDir}"
    rename { it.replace(assembly, allInOne) }
}```